pkg/cli/cmds/server.go

```diff
@@ -210,6 +219,33 @@ func NewServerCommand(action func(*cli.Context) error) *cli.Command {
 				Destination: &ServerConfig.DatastoreKeyFile,
 				EnvVars:     []string{version.ProgramUpper + "_DATASTORE_KEYFILE"},
 			},
+			&cli.StringFlag{
+				Name:        "etcd-snapshot-restore-path",
+				Usage:       "(db) Path to snapshot file to be restored",
+				Destination: &ServerConfig.EtcdRestorePath,
+			},
+			&cli.BoolFlag{
+				Name:        "etcd-disable-snapshots",
+				Usage:       "(db) Disable automatic etcd snapshots",
+				Destination: &ServerConfig.EtcdDisableSnapshots,
+			},
+			&cli.DurationFlag{
+				Name:        "etcd-snapshot-interval",
+				Usage:       "(db) Snapshot interval time",
+				Destination: &ServerConfig.EtcdSnapshotInterval,
+				Value:       defaultSnapshotIntervalMinutes * time.Minute,
+			},
+			&cli.IntFlag{
+				Name:        "etcd-snapshot-retention",
+				Usage:       "(db) Number of snapshots to retain",
+				Destination: &ServerConfig.EtcdSnapshotRetention,
+				Value:       defaultSnapshotRentention,
+			},
+			&cli.StringFlag{
+				Name:        "etcd-snapshot-dir",
+				Usage:       "(db) Directory to save db snapshots",
```

Comment:
I think we should say something like `(Default location: ${data-dr}/db/snapshots)`